data/scripting/messages.lua

```diff
@@ -82,20 +87,44 @@ function message_box(player, title, body, parameters)
 end
 
 -- RST
--- .. function:: campaign_message_box(message, [sleeptime])
+-- .. function:: campaign_message_box({message, [sleeptime, parameters]})
 --
---    Sets message.h and message.w if not set and calls
---    message_box(player, title, body, parameters) for player 1
+--    Sets the width and height of the message box and calls
+--    :meth:`message_box` for player 1. This function expects a table, so surround
+--    all values in a pair of curly braces.
 --
---    :arg message: the message to be sent
+--    :arg message: the message consist of the `title` and the `body`. Note that the `body`
+--                  must be formatted using the :ref:`richtext functions <richtext.lua>`.
 --    :arg sleeptime: ms spent sleeping after the message has been dismissed by the player
 --
---    Besides the normal message arguments (see wl.Game.Player:message_box) the following ones can be used:
---
---    :arg position: A string that indicates at which border of the screen the message box shall appear. Can be "top", "bottom", "right", "left" or a combination (e.g. "topright"). Overrides posx and posy. Default: Center. If only one direction is indicated, the other one stays central.
---    :arg scroll_back: If true, the view scrolls/jumps back to where it came from. If false, the new location stays on the screen when the message box is closed. Default: False.
---    :arg show_instantly: If true, the message box is shown immediately. If false, this function will wait until the player leaves the roadbuilding mode. Use this with care because it can be very interruptive. Default: false.
+--    Besides the normal message parameters (see :meth:`wl.game.Player.message_box`) the following ones can be used:
+--
+--    :arg position: A string that indicates at which border of the screen the message box shall appear.
+--                   Can be "top", "bottom", "right", "left" or a combination (e.g. "topright").
+--                   Overrides posx and posy. Default: Center. If only one direction is indicated,
+--                   the other one stays centered.
+--    :arg scroll_back: If true, the view scrolls/jumps back to where it came from. If false, the new
+--                      location stays on the screen when the message box is closed. Default: False.
+--    :arg show_instantly: If true, the message box is shown immediately. If false, this function will
+--                    wait until the player leaves the roadbuilding mode. Use this with care because it
+--                    can be very interruptive. Default: false.
+--
+--    Example:
+-- .. code-block:: lua
+--
+--    campaign_message_box({title = "The title",    -- part of message
+--                          body = p("The body"),   -- part of message
+--                          200,                    -- optional sleeptime
```

Comment:
Are you sure that this will work?

Reply:
I have tested this with the debug console.

Reply:
`campaign_message_box({title="AAA",body="BBB",1000});print("XXX")` prints the message to the console directly after closing the box. I think there must be a key missing somewhere.

Reply:
Or `sleeptime` does not belong in the curly braces?

Reply:
Yes, you are right :-) I am fiddling with with REST markup right now to have a meaningful look in the documentation.